ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadV2Test.java

```diff
@@ -87,7 +87,9 @@ public void shouldReturnValidIfWithdrawalsIsNotNull_WhenWithdrawalsAllowed() {
         .when(blockchain.getBlockHeader(mockHeader.getParentHash()))
         .thenReturn(Optional.of(mock(BlockHeader.class)));
     var resp =
-        resp(mockEnginePayload(mockHeader, Collections.emptyList(), withdrawalsParam, null, null));
+        resp(
```

Comment:
This seems like a bad signal that we had to update older tests. Probably a number of different ways to control this better, should EnginePayloadParameter get extended? Versioned?

Reply:
fair point. have not addressed this as yet. open to suggestions on the best approach.

Reply:
also think this can be done in a separate PR